kritis-charts/values.yaml

```diff
@@ -7,7 +7,7 @@ replicaCount: 1
 image:
   image: kritis-server
   # TODO(tstromberg): Have a versioned tag after release.
```

Comment:
We could probably get rid of this TODO, @tstromberg can confirm!

Reply:
will remove.